postgresql/PostgreSqlPreferences.js

```diff
@@ -9,6 +9,26 @@ class PostgreSqlPreferences extends DbPreferences {
   constructor() {
     super("db.postgresql");
   }
+
+  /**
+   * Connection options
+   *
+   * @return {object}
+   */
+  getConnOptions() {
```

Comment:
is it possible to resuse the superclass method instead of copying the whole one from that?
eg.:
```
getConnOptions() {
    const options = super.getConnOptions().options;
    options["ssl"] = app.preferences.get(this.connPrefOptKeyPrefix + ".ssl");
    
    return options;
  }
```

Reply:
or:
```
getConnOptions() {
    return Object.assign({options: {ssl: app.preferences.get(this.connPrefOptKeyPrefix + ".ssl")}}, super.getConnOptions());
  }
```